### PR TITLE
209

### PR DESCRIPTION
**Pull request by mofsesam**
Would like some comments if it is in the right "ball-park".

### CODE REVIEW

File: training/040_projects_and_infrastructure/020_Beginner.rst [hunk 1]
```diff
@@ -34,8 +36,116 @@ Lage grupp/tilganger
...
+wipe dev-node
+restart dev-node
+run sesam *test* command. This will do an upload (of checked out config in project folder), a *run* and a *verify*.
+look at output from *verify*. If you expect the new config to produce all the same output/endpoint data as previous configs, *verify* should return that all tests passed.
+If output is not the same you need to look for bugs in your config. If you expected output to change, you need to check if new output is as you expected. If you use command *update*, you will download
+the current output values from your dev-node to your expected folder. You can than use diff functionallity in you version controll system to check differenses from previous expect values. If this is as you intended
+you can make this the new expected values by staging/commiting changes to expected to your project. If not you can revert to old expected files.
+
+Example with bash, git and sesam-py:
+::
```
Comment (ga-hegsvold) [inline training/040_projects_and_infrastructure/020_Beginner.rst:126]: Add a line break here to get valid formatting.

File: training/040_projects_and_infrastructure/020_Beginner.rst [hunk 2]
```diff
@@ -34,8 +36,116 @@ Lage grupp/tilganger
...
+    cd your-sesam-project
+    git status
+    git checkout my-feature
+    cd node/
+    sesam wipe -vv
+    sesam restart -vv
+    sesam test -vv -scheduler-max-run-time 3000 -print-scheduler-log
```
Comment (ga-hegsvold) [inline training/040_projects_and_infrastructure/020_Beginner.rst:133]: Note that the sesam client expects options first, then a command:

`sesam [options] command`

so put `test`, `wipe`, etc at the end.

**Review by ga-hegsvold — COMMENTED** [review]
I think this is in the right ball-park. :-)
The sesam client is also described here: file:///home/gah/git-repos/datanav/docs/_build/html/sesam-client.html

Maybe consider condensing this section or focus on stuff that has not been properly covered in the other section. And link to it.

**Comment by BaardBouvet** [comment]
This is just noise as the training branch is dead

**Pull request closed without merging**